### ISSUE TITLE
Add tooltip to inputs with validation errors

### ISSUE THREAD

**Issue by natanaelsirqueira** [diego3g/rocketredis#77]
The form validation errors are not shown anywhere. It seems like we're missing a tooltip on the input icon.

![tooltip](https://user-images.githubusercontent.com/13697898/87855503-c9afdb00-c8e6-11ea-9995-1306394c1882.gif)

**Comment by adarshsaraogi** [comment]
@natanaelsirqueira yeah even i came up with the same impression, we should implement with the proper sentence,
what do you think?

**Comment by natanaelsirqueira** [comment]
Sorry, I'm not sure what you meant.

**Comment by adarshsaraogi** [comment]
@natanaelsirqueira we should add the validation error using tooltip.

**Comment by natanaelsirqueira** [comment]
Oh, yes. I think it is a good call, at least it's the UX my brain "asked" when I tried the feature.

**Comment by diego3g** [comment]
I'll create the layout for this.
